cves/kernel/CVE-2015-8967.yml

```diff
@@ -386,22 +429,29 @@ lessons:
 
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
-  defense_in_depth:
-    applies:
-    note:
+  defense_in_depth: 
+    applies: true
+    note: |
+      The vulnerability demonstrates the importance of implementing multiple layers of defense.
+      While this vulnerability is related to privileg eesclation, implementing defense in depth
+      principles can help mitigate other potential risks and reduces the impact of similar vulnerabilities.
```

Comment:
I agree Defense in Depth applies, but could also go a little deeper in analysis. The fix makes the data structure immutable which in the context of Defense in Depth, is a pretty "deep" security fix. Just a thought